Solutions/OracleDatabaseAudit/Data/system_generated_metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "Name": "OracleDatabaseAudit",
+  "Author": "Microsoft - support@microsoft.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/oracle_logo.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The Oracle Database Audit solution provides the capability to ingest [Oracle Database](https://www.oracle.com/database/technologies/) audit events into Microsoft Sentinel through the syslog. Refer to [documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/introduction-to-auditing.html#GUID-94381464-53A3-421B-8F13-BD171C867405) for more information.\r\n  \r\n  **Underlying Microsoft Technologies used:** \r\n\r\n This solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:\r\n\n  a. [Agent-based log collection (Syslog)](https://docs.microsoft.com/azure/sentinel/connect-syslog)\r\n\n\n\n",
+  "Metadata": "SolutionMetadata.json",
+  "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\OracleDatabaseAudit",
+  "TemplateSpec": true,
+  "Is1PConnector": false,
+  "Version": "3.0.0",
+  "publisherId": "azuresentinel",
+  "offerId": "azure-sentinel-solution-oracledbaudit",
+  "providers": [
+    "Oracle"
+  ],
+  "categories": {
+    "domains": [
+      "Application"
+    ]
+  },
+  "firstPublishDate": "2021-11-05",
+  "support": {
+    "name": "Microsoft Corporation",
+    "email": "support@microsoft.com",
+    "tier": "Microsoft",
+    "link": "https://support.microsoft.com"
+  },
+  "Data Connectors": "[\n  \"Data Connectors/Connector_OracleDatabaseAudit.json\"\n]",
+  "Parsers": "[\n  \"OracleDatabaseAuditEvent.txt\"\n]",
```

Comment:
Please update the parser extension to yaml and update with same in data file.  Once changes added in data file, please repackage the solution.

Reply:
Completed these changes and JFYI, didn't add as this PR mainly  content review once done ,thought of doing these changes,pushed now